app/src/main/java/com/codesoom/assignment/AppHttpHandler.java

```diff
@@ -0,0 +1,266 @@
+        logger.info("method : " + method + " /  uriPath : " + uriPath + " / body : " + body);
+
+        HttpStatusCode httpStatusCode = HttpStatusCode.OK;
+        try {
+            if (taskPath.equals(rootPath)) {
+                RequestProcessEntity requestProcessEntity = new RequestProcessEntity
+                        .RequestProcessEntityBuilder()
+                        .setBody(body)
+                        .setMethod(method)
+                        .setTaskId(taskId).build();
+                AppResponseEntity appResponseEntity = requestProcessor(requestProcessEntity);
+                contents = appResponseEntity.getContents();
+                httpStatusCode = appResponseEntity.getHttpStatusCode();
+            }
+        } catch (NotFoundTaskIdException ne) {
+            httpStatusCode = HttpStatusCode.NOT_FOUND;
+        }
+
+
+        exchange.sendResponseHeaders(httpStatusCode.getStatusCode(), contents.getBytes().length);
+        OutputStream outputStream = exchange.getResponseBody();
+        outputStream.write(contents.getBytes());
+        outputStream.flush();
+        outputStream.close();
+    }
+
+    /**
+     * GET, POST, PUT, DELETE 에 대한 요청에 대한 HttpStatusCode 값과  ResponseBody 리턴
+     * @param requestProcessEntity
+     * @return AppResponseEntity
+     * @throws IOException
+     */
+    private AppResponseEntity requestProcessor(RequestProcessEntity requestProcessEntity) throws IOException {
+        String contents = "";
+        HttpStatusCode httpStatusCode = HttpStatusCode.OK;
+        String taskId = requestProcessEntity.getTaskId();
+        String body = requestProcessEntity.getBody();
+        String method = requestProcessEntity.getMethod();
+        boolean isBody = requestProcessEntity.isBodyBlank();
+        HttpMethod httpMethod = HttpMethod.valueOf(method);
+
+        switch (httpMethod) {
+            case GET:
+                contents = tasksToJSONByPath(taskId);
+                httpStatusCode = HttpStatusCode.OK;
+                break;
+            case POST:
+                if (isBody) {
+                    break;
+                }
+
+                Task task = toTask(body);
+                task.setId(getTaskMaxId());
+                tasks.add(task);
+                httpStatusCode = HttpStatusCode.CREATED;
+                contents = tasksToJSON(task);
+                break;
+            case DELETE:
+                deleteTask(taskId);
+                httpStatusCode = HttpStatusCode.NO_CONTENT;
+                break;
+            case PUT:
+                contents = modifyTask(requestProcessEntity);
+                httpStatusCode = HttpStatusCode.OK;
+                break;
+            default:
+                break;
+        }
+        return new AppResponseEntity(httpStatusCode, contents);
+    }
+
+    /**
+     *  Task ID에 해당하는 내용을 수정
+     * @param requestProcessEntity
+     * @return String(JSON)
+     * @throws IOException
+     */
+
```

Comment:
JavaDoc summary는 완성된 문장으로 작성하고, 입력 파라미터와 리턴값을 강하게 의식하며 작성하도록 합니다. 마침표는 꼭 찍어주세요. 

다음은 제가 작성해본 것이에요.
```suggestion
    /**
     *  할 일 수정 요청을 받아 할 일을 수정하고, JSON 문자열로 리턴합니다.
     * @param requestProcessEntity 할 일 수정 요청
     * @return String JSON 형식의 할 일 정보
     */
```

이 주석을 읽어보면 어딘가가 아쉬운 면이 느껴질 것입니다. 예를 들어 리턴 타입이라던가, 입력 파라미터의 타입이라던가...